app/experimenter/experiments/api/v5/serializers.py

```diff
@@ -156,6 +156,14 @@ def _validate_feature_configs(self, data):
 
         return data
 
+    def create(self, validated_data):
+        experiment = super().create(validated_data)
+        experiment.reference_branch = NimbusBranch.objects.create(
+            experiment=experiment, name="control", feature_enabled=False
```

Comment:
Shouldn't you set slug here?  Or are you relying on the fact that they'll have to set the description, which will force them to save, which will then generate the slug?

Reply:
Setting the slug requires a bunch of other edits: https://github.com/mozilla/experimenter/pull/5315#discussion_r640941887